index.js

```diff
@@ -49,8 +54,9 @@ const termlog = (options = {}) => {
 
   const ws = new WebSocket(`${options.ssl ? "wss" : "ws"}://${options.host}:${options.port}`);
 
+  configure(ws, defaultConsole);
```

Comment:
in which case this is required? Since the `configure` makes the log send via WebSocket and it's only possible once the WebSocket is open, I don't see why we need to move it here.

Reply:
As the opening of the WebSocket is running async it may or may not happen that messages get logged to the browser console before the connection is established. Only after connecting the console is configured.
When running configure beforehand the messages logged before a connection happens are processed by the sendWhenConnected function until the connection is made or the timeout is hit.